src/app/modules/item/components/item-remove-button/item-remove-button.component.ts

```diff
@@ -15,6 +15,8 @@ import { Router } from '@angular/router';
   styleUrls: [ './item-remove-button.component.scss' ],
 })
 export class ItemRemoveButtonComponent implements OnChanges, OnDestroy {
+  @Output() removeApprove = new EventEmitter<void>();
```

Comment:
I think `removalApproved` or `approveRemoval` is more suited, same for the other component

Reply:
Fixed